src/main/java/com/github/princesslana/slothbot/MessageCounter.java

```diff
@@ -81,6 +106,53 @@ public void clearBuckets(String channelId) {
                 .collect(ImmutableList.toImmutableList()));
   }
 
+  public boolean getBotCounterConfig(String guildId) {
```

Comment:
I think this is better named something like `isCountingBotMessages`

Reply:
Agreed, I'll change to that.